### PR TITLE
fix(WebexMeeting): resize dark and light loading logos

### PR DESCRIPTION
**Pull request by lungancatalin**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-267914

### CODE REVIEW

File: src/components/WebexMeeting/WebexMeeting.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
   }
 
   .loading-logo {
-    background: var(--wxc-loading-logo) no-repeat center / 6.5rem 4.627rem;
+    background: var(--wxc-loading-logo) no-repeat center / 11.6875rem;
```
Comment (lalli-flores) [inline src/components/WebexMeeting/WebexMeeting.scss:30]: Can you make this a variable or formulate as to how you arrived to this number?

Comment (lungancatalin) [reply src/components/WebexMeeting/WebexMeeting.scss:30]: Hi @lalli-flores,

I received in Jira, from Hannah, the correct dimensions for loading the logo, which should be 187 * 187 px. I transformed these pixels into rem 187/16 = 11.6875 and used the size in rem. That's it. I will add a new variable for that dimension and add it in both themes. Thank you.

Comment (lalli-flores) [reply src/components/WebexMeeting/WebexMeeting.scss]: Thanks for the clarification @lungancatalin. Someone without access or knowledge of the Figma spec would look at the number and wonder where this seemingly random number came from.

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**